dockerfiles/steps/git-fetch-metadata.bash

```diff
@@ -15,13 +17,15 @@ fi
 try fetch-update-meta "$IO_FETCH_META/.git" "$IO_FETCH_META/modules" "$IO_FETCH_META/collections" "$ARG_GIT_REF" "$IO_FETCH_META/canonical.json"
 try rm -rf "$IO_FETCH_META/.git"
 
-[[ -f $(dirname "$IO_RESOURCES")/resources ]] || {
-    say "Expected $(dirname "$IO_RESOURCES")/resources directory to exist"
-}
+export HACK_CNX_LOOSENESS=1
```

Comment:
Should this be disabled?

Reply:
I'd like to change this one _right after_ a version of enki is tagged with these changes in it.